packages/p2p-v2/src/pages/orders/components/ChatError/ChatError.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { MouseEventHandler } from 'react';
+import { Button, Text, useDevice } from '@deriv-com/ui';
+
+type TChatErrorProps = {
+    onClickRetry: MouseEventHandler<HTMLButtonElement>;
+};
+
+const ChatError = ({ onClickRetry }: TChatErrorProps) => {
+    const { isMobile } = useDevice();
+
+    return (
+        <div className='flex flex-col gap-[1.6rem] items-center'>
+            <Text size={isMobile ? 'lg' : 'md'}>Oops, something went wrong</Text>
```

Comment:
[suggestion]: for small css changes, we can move away from using a JS solution such as isMobile.

you can use this hook for UIs with different layout/elements on desktop and mobile.